openshift/ci-operator/generate-ci-config.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+branch=${1-'openshift-serverless-v1.0.0'}
```

Comment:
I'm not sure about this name. Please let me know if this should be changed. For core knative image we use "knative-v0.7" and

Reply:
The CSV is using `serverless-operator.v1.0.0` but I'm not sure what the correct convention is.

Reply:
This name determines where the image will be available for tests in other openshift repositories (in case we ever need it). The image is promoted there after a PR is merged. 
The current location would then be: `registry.svc.ci.openshift.org/openshift/openshift-serverless-v1.0.0:serverless-operator` (this is the same scheme that we use for e.g. knative serving: `registry.svc.ci.openshift.org/openshift/knative-v0.7.1:knative-serving-activator`)
I think this scheme is alright.
If we used `serverless-operator.v1.0.0` there it would look like this: `registry.svc.ci.openshift.org/openshift/serverless-operator.v1.0.0:serverless-operator`